docs/src/componentDocs/Header/markdown/HeaderExamples.mdx

```diff
@@ -1 +1,189 @@
+
+<Box
+  sx={{
+    width: '100%',
+    textAlign: 'center',
+  }}>
+  <Box
+    sx={{
+      width: '100%',
+      maxWidth: '500px',
+    }}
+    component="img"
+    alt="Header"
+    src={headerImageExample}
+  />
+</Box>
+
+```
+<Header
+    title={'Valley Forge'}
+    subtitle={'The Last Stand'}
+/>;
+```
+
+## Header with Background image
+
+<Box
+  sx={{
+    width: '100%',
+    textAlign: 'center',
+  }}>
+  <Box
+    sx={{
+      width: '100%',
+      maxWidth: '500px',
+    }}
+    component="img"
+    alt="Header background image"
+    src={headerBackgroundExample}
+  />
+</Box>
+
+```
+<Header
+    title={'Valley Forge'}
+    subtitle={'The Last Stand'}
+    backgroundImage={farm.jpg}
+/>;
+```
+
+## Header with Actions and Menu
+
+<Box
+  sx={{
+    width: '100%',
+    textAlign: 'center',
+  }}>
+  <Box
+    sx={{
+      width: '100%',
+      maxWidth: '500px',
+    }}
+    component="img"
+    alt="Header actions"
+    src={headerActionsExample}
+  />
+</Box>
+
+```
+<Header
+    title={'Valley Forge'}
+    subtitle={'The Last Stand'}
+    actionItems={[
+        {
+            icon: { name: 'settings'},
+            onPress: () => {},
+        },
+        {
+            icon: { name: 'more-vert' },
+            onPress: () => {},
+        },
+    ]}
+    icon={{ name: 'menu' }}
+/>;
+```
+
+## Header with Search
+
+<Box
+  sx={{
+    width: '100%',
+    textAlign: 'center',
+  }}>
+  <Box
+    sx={{
+      width: '100%',
+      maxWidth: '500px',
+    }}
+    component="img"
+    alt="Header search"
+    src={headerSearchExample}
+  />
+</Box>
+
+```
+<Header
+    title={'Valley Forge'}
+    subtitle={'The Last Stand'}
+    actionItems={[
+        {
+            icon: { name: 'settings'},
+            onPress: () => {},
+        },
+        {
+            icon: { name: 'more-vert' },
+            onPress: () => {},
+        },
+    ]}
+    icon={{ name: 'menu' }}
+    searchableConfig={{ onChangeText: () => {} }}
+/>;
+```
+
+## Header with Collapsing and Expanding
+
+<Box
+  sx={{
+    display: 'flex',
+    justifyContent: 'space-evenly',
+    alignItems: 'center',
+    mb: 2,
+    width: '100%',
+    textAlign: 'center',
+  }}>
+  <Box
+    sx={{
+      width: '40%',
+    }}
+    component="img"
+    alt="Header collapsed"
+    src={headerCollapsedExample}
+  />
+  <Box
+    sx={{
+      width: '40%',
+    }}
+    component="img"
+    alt="Header expanded"
+    src={headerExpandedExample}
+  />
+</Box>
+
+```
+<Header
+    title={'Valley Forge'}
+    subtitle={'The Last Stand'}
+    info={'Information'}
+    backgroundImage={farm.jpg}
+    icon={{ name: 'menu' }}
+    onIconPress={() => {}}
+    actionItems={[
+        {
+            icon: { name: 'settings'},
+            onPress: () => {},
+        },
+        {
+            icon: { name: 'more-vert' },
+            onPress: () => {},
+        },
+    ]}
+    variant="static"
+    icon={{ name: 'menu' }}
+    searchableConfig={{ onChangeText: () => {} }}
```

Comment:
You can leave off all of this stuff that is not related to the expand/collapse behavior we are demonstrating in this example.